BasicPlaybackSampleApp/BasicPlaybackSampleApp/Players/BasicSimplePlayerViewController.m

```diff
@@ -95,9 +96,34 @@ - (void)viewDidLoad {
   // Attach it to current view
   [self addPlayerViewController:self.ooyalaPlayerViewController];
   
+  __weak typeof(self) weakSelf = self;
+  //OS: to avoid calling '[weakSelf.player play]' until player really ready to play
+  void (^expectedBlock) (OOVideo *currentItem);
+  expectedBlock = ^(OOVideo *currentItem) {
```

Comment:
The same about block as in `OoyalaSkinSDK`

Reply:
I added answer in **native-skin** repo